### PR TITLE
brew-bundle: add page

### PR DESCRIPTION
**Pull request by melgu**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/brew-bundle.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# brew bundle
+
+> Bundler for Homebrew, Homebrew Cask and the Mac App Store.
+> More information: <https://github.com/Homebrew/homebrew-bundle>.
+
+- Install packages from Brewfile at current path:
```
Comment (agnivade) [inline pages/osx/brew-bundle.md]: at _the_ current path

File: pages/osx/brew-bundle.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`brew bundle`
+
+- Install packages from Brewfile at specific path:
```
Comment (agnivade) [inline pages/osx/brew-bundle.md]: from a specific Brewfile

File: pages/osx/brew-bundle.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`brew bundle --file={{path/to/file}}`
+
+- Create Brewfile from all installed packages:
+
+`brew bundle dump`
+
+- Uninstall all formulae not listed in Brewfile:
```
Comment (agnivade) [inline pages/osx/brew-bundle.md]: in _the_ Brewfile here and after

**Comment by melgu** [comment]
Thanks for the feedback.

**Pull request merged**